app/controllers/responsible_body/devices/who_to_contact_controller.rb

```diff
@@ -18,8 +18,8 @@ def create
       render :new, status: :unprocessable_entity
     else
       chosen_contact = @form.chosen_contact
-      chosen_contact.save! unless chosen_contact.persisted?
-      @school.preorder_information.update!(school_contact: chosen_contact)
+      chosen_contact.save!
+      @school.preorder_information&.update!(school_contact: chosen_contact)
```

Comment:
My test school didn't have a preorder. Should I have given it one? Otherwise this will fix potentially an issue

Reply:
Yeah, you won't be able to get to this screen without a `PreorderInformation`.

Reply:
ok, I've added preorder info on the test and removed the guard